src/apps/backend/modules/comment/rest_api/comment_routers.py

```diff
@@ -0,0 +1,16 @@
+from flask import Blueprint
+
+from modules.comment.rest_api.comment_view import CommentView
+
+
+class CommentRouter:
+    @staticmethod
+    def create_route(*, blueprint: Blueprint) -> Blueprint:
+        blueprint.add_url_rule("/comments", view_func=CommentView.as_view("create_comment"), methods=["POST"])
+        blueprint.add_url_rule(
+            "/comments/<task_id>", view_func=CommentView.as_view("get_comments_by_task"), methods=["GET"]
```

Comment:
This is not CRUD.

CRUD are:

GET `/comments`
GET `/comments/:commentId`
PATCH
PUT
DELETE
...

In order to fetch comments for a task, it should be:

GET `/comments?task_id=?` OR `GET /tasks/:taskId/comments`